### ISSUE TITLE
Download Original JPG link is problematic

### ISSUE THREAD

**Issue by jvine** [sul-dlss/earthworks#649]
In the Downloads menu, the TIFF downloads, and the JPG opens the image in the current window, leaving the user without any context but the back button. 

I think the answer is no... but is there any way to force the image to download?

**Comment by mejackreed** [comment]
Can you provide an example link to help troubleshoot?

**Comment by jvine** [comment]
https://earthworks.stanford.edu/catalog/princeton-w3763818m

**Comment by mejackreed** [comment]
I think maybe the answer is no. Open to others suggestions though

**Comment by cbeer** [comment]
https://caniuse.com/#feat=download 🤷‍♂️

**Comment by mejackreed** [comment]
So maybe a feature in GeoBlacklight that creates this link and then adds the `download` attribute if it doesn't exist?

**Comment by jvine** [comment]
Except that `download` doesn't seem to actually work, and doesn't tell the screenreader anything either. Including the word "Download" in the link text #646 will help IRL, but .. ugh.
